target/linux/ramips/image/mt7621.mk

```diff
@@ -1500,6 +1500,16 @@ define Device/sercomm_na502s
 endef
 TARGET_DEVICES += sercomm_na502s
 
+define Device/snr_snr-cpe-me2-lite
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := SNR
+  DEVICE_MODEL := SNR-CPE-ME2-Lite
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e kmod-mt7663-firmware-ap -wpad-basic-wolfssl -kmod-mt7615-firmware
```

Comment:
Do I need to specify `wpad-basic-wolfssl`? It seems to be included in the package list by default.
Maybe you should enable `uboot-envtools`? Remove r/o for uboot-env in dts and set region in `package/boot/uboot-envtools/files/ramips`?

Reply:
This device has wifi, you should not remove `wpad-basic-wolfssl`.

`uboot-envtools` would be nice too, then you have to add a configuration to `package/boot/uboot-envtools/files/ramips`.